output/failed_runs.txt

```diff
@@ -0,0 +1,2 @@
+hate-speech-detection-hf-hf-meta-model-nlp_hf_meta-model-pipeline
+hate-speech-detection-hf-hf-meta-model-nlp_hf_meta-model-pipeline
```

Comment:
could you remove this first and then can we add it to gitignore? sorry, I should have done it.

Reply:
Done!